packages/mobile/src/components/leaving-audius-drawer/LeavingAudiusDrawer.tsx

```diff
@@ -40,11 +38,7 @@ export const LeavingAudiusDrawer = () => {
     >
       <View style={styles.root}>
         <Text>{messages.content}</Text>
-        <HelpCallout
-          numberOfLines={10}
```

Comment:
is this loss of information relevant? Hopefully the new one is appropriately sized based on its content

Reply:
naw i dont think so